spec/games_spec.rb

```diff
@@ -61,4 +61,29 @@
       expect(game_report).to eq(expected_final_report)
     end
   end
+
+  describe '#death_report' do
+    subject(:games){described_class.new('spec/fixtures/example.log')}
```

Comment:
death_report should be your subject since it is your point of interest in these tests.